### DISCUSSION TITLE
Reset Interface mode in custom script

### DISCUSSION THREAD

**Discussion by PaulR282** [netbox-community/netbox#16953] [Help Wanted!] [answered]
Hi,

I want to reset the interface mode in a custom script. If I do `interface.mode = None` I get the error message `IntegrityError: null value in column "mode" of relation "dcim_interface" violates not-null constraint DETAIL: [...]`. 
Resetting e.g. the untagged VLAN through `interface.untagged_vlan = None` works.

**Comment by zmaster7** [comment, marked as answer]
If you clear the interface mode value via the UI, the changelog indicates the value is being changed back to "" rather than null/None. I would try changing it to an empty string value and see if that works for you instead.

**Reply by PaulR282** [reply]
That works, thanks.
